packages/appsync-emulator-serverless/schemaWrapper.js

```diff
@@ -1,7 +1,27 @@
 const GraphQLJSON = require('graphql-type-json');
 
+const {
+  GraphQLDate,
+  GraphQLTime,
+  GraphQLDateTime,
+} = require('graphql-iso-date');
+
+const {
+  EmailAddress,
+  RegularExpression,
+  URL,
+} = require('@okgrow/graphql-scalars');
+
+const GraphQLPhone = new RegularExpression('GraphQLPhone', /^.*$/);
```

Comment:
I think we can come up with something better here.

Reply:
Rather than the regular expression?

Reply:
The current implementation will accept _any_ string.  I am thinking we should use a regular expression like [this](https://stackoverflow.com/questions/2113908/what-regular-expression-will-match-valid-international-phone-numbers) and testing a US phone number, along with a phone number in another country.

Reply:
Gotcha. Unfortunately, a single regex is going to be really complex to validate all possible phone combos. Perhaps a library is a better way to go. That's why I just committed a change for this scalar to use `graphql-phone-type`